app/_hub/kong-inc/standard-webhooks/overview/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+nav_title: Overview
+title: Overview
+---
+
+The Standard Webhooks plugin lets you validate incoming webhooks using the [Standard Webhooks](https://github.com/standard-webhooks/standard-webhooks) specification. 
+Kong is a contributor to this specification.
```

Comment:
```suggestion
The Standard Webhooks plugin lets you validate incoming webhooks using the [Standard Webhooks](https://github.com/standard-webhooks/standard-webhooks) specification. 
Kong contributes to this specification.
```
I think maybe a sentence that explains who this might be for or what use case(s) this supports would help me understand if this plugin is for me.